Modules/mathmodule.c

```diff
@@ -1338,19 +1352,25 @@ FUNC2(remainder, m_remainder,
       "y, the nearest even value of n is used. The result is always exact.")
 FUNC1(sin, sin, 0,
       "sin($module, x, /)\n--\n\n"
-      "Return the sine of x (measured in radians).")
+      "Return the sine of x (measured in radians).",
+      "math domain error")
 FUNC1(sinh, sinh, 1,
       "sinh($module, x, /)\n--\n\n"
-      "Return the hyperbolic sine of x.")
+      "Return the hyperbolic sine of x.",
+      "math domain error")
 FUNC1(sqrt, sqrt, 0,
       "sqrt($module, x, /)\n--\n\n"
-      "Return the square root of x.")
+      "Return the square root of x.",
+      "math.sqrt() expects a non-negative input, got '%R'.\n"
```

Comment:
Super-nit: can we please go with "nonnegative" rather than "non-negative". (Both are acceptable in theory, but the non-hyphenated version is winning the wars these days, and as a mathematician, "non-negative" looks strange to my eyes.)

Related: https://math.stackexchange.com/a/3344027/117283

Reply:
Thanks for your professional advice! I noticed that there are also multiple places in the `mathmodule.c` where the non-hyphen version is used, including non-zero, non-int, non-finite, etc. Maybe we could modify them gradually :)